docs/specification.md

```diff
@@ -6,7 +6,7 @@ sidebar_label: Specification
 
 An example of how using the all-contributors spec table can recognize all contributors
 <div align="center">
-    <img src="../assets/contributors-table-small.png" alt="Contributors Table Screenshot" width="800px" />
+    <img src="https://raw.githubusercontent.com/all-contributors/all-contributors/master/docs/assets/contributors-table-small.png" alt="Contributors Table Screenshot" width="800px" />
```

Comment:
Is it necessary to use the RAW GH URL for this?

Reply:
using raw gh url will make it visible on both the markdown and deployed version.

Reply:
Good point.